lntest/itest/lnd_channel-funding_fund_max_test.go

```diff
@@ -294,3 +329,25 @@ func sweepNodeWalletAndAssert(net *lntest.NetworkHarness, ht *harnessTest,
 	// Make sure all funds were swept.
 	checkChannelBalance(ht, node, 0)
 }
+
+// Helper closure to assert the proper response to a channel balance RPC.
+func checkWalletBalance(ht *harnessTest, node lnrpc.LightningClient,
```

Comment:
I think we also have this assertion already somewhere.

Reply:
I've removed `checkWalletBalance` and `checkChannelBalance` from this test and added the functions to `assertions.go` and call those instead.